index.bs

```diff
@@ -4684,16 +4739,24 @@ An interface |A| is considered to be a
     For example, if author 1 writes:
 
     <pre highlight="webidl">
+        [Global,Exposed=Window]
```

Comment:
I suggest we require an identifier since omitting it would only benefit the Window case as far as I can tell and it seems better to be explicit for these.